# This file is used for puting objects information into the system.
# This file surppports multiple objects.
number_objects: 1
# Positions of the objects (m)
position:
- [0.0, -2.0, -0.43]
# Constrant angles of the objects (rad)
constrant_angle: [0.17]
# Attitude angles of the objects (rad)
attitude:
- [0.0,  0.0,  0.0]
